ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -164,7 +168,18 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
             pressure_size, displacement_size>::Zero(pressure_size,
                                                     displacement_size);
 
+    MaterialLib::Solids::MechanicsBase<DisplacementDim> const& solid_material =
+        *_process_data.solid_materials[0];
+
+    auto linear_elastic_mp =
+        static_cast<MaterialLib::Solids::LinearElasticIsotropic<
+            DisplacementDim> const&>(solid_material)
+            .getMaterialProperties();
+
     double const& dt = _process_data.dt;
+    double const R_s = _process_data.specific_gas_constant;
+    double const T = _process_data.reference_temperature;
```

Comment:
Maybe `T_ref` to make clear later on that a reference temperature is meant?

Reply:
Yes that's better